core/trino-main/src/main/java/io/trino/type/ArrayParametricType.java

```diff
@@ -39,13 +39,13 @@ public String getName()
     }
 
     @Override
-    public Type createType(TypeManager typeManager, List<TypeParameter> parameters)
```

Comment:
The `TypeParameter` has the advantage that it's "resolved" version of the `TypeSignatureParameter`. From API perspective this might be better: the `ParametricType#createType` implementation gets something that's more processed and ready to use

Reply:
Yes, I'm aware, but this is a very minor advantage. Besides, the method already takes a TypeManager, whose purpose is to resolve types.